src/V3Task.cpp

```diff
@@ -1316,6 +1316,9 @@ class TaskVisitor final : public VNVisitor {
             }
         }
 
+        // Mark the fact that this function allocates std::process
+        if (nodep->isFromStd() && nodep->name() == "self") cfuncp->setNeedProcess();
```

Comment:
Do we need it this early?

Reply:
We cannot have it any later. This is the last moment when we have the access to the original function name.

Notice that the name is checked on `nodep` and the flag is saved on `cfuncp`.